modules/apps/27-interchain-accounts/host/keeper/keeper_test.go

```diff
@@ -104,41 +106,51 @@ func TestKeeperTestSuite(t *testing.T) {
 }
 
 func (suite *KeeperTestSuite) TestIsBound() {
-	isBound := suite.chainA.GetSimApp().ICAKeeper.IsBound(suite.chainA.GetContext(), types.PortID)
+	suite.SetupTest()
+
+	path := NewICAPath(suite.chainA, suite.chainB)
+	suite.coordinator.SetupConnections(path)
+
+	err := SetupICAPath(path, TestOwnerAddress)
+	suite.Require().NoError(err)
+
+	isBound := suite.chainB.GetSimApp().ICAHostKeeper.IsBound(suite.chainB.GetContext(), path.EndpointB.ChannelConfig.PortID)
 	suite.Require().True(isBound)
 }
 
 func (suite *KeeperTestSuite) TestGetAllPorts() {
 	suite.SetupTest()
+
 	path := NewICAPath(suite.chainA, suite.chainB)
 	suite.coordinator.SetupConnections(path)
 
 	err := SetupICAPath(path, TestOwnerAddress)
 	suite.Require().NoError(err)
 
-	expectedPorts := []string{types.PortID, TestPortID}
+	expectedPorts := []string{types.PortID}
 
-	ports := suite.chainA.GetSimApp().ICAKeeper.GetAllPorts(suite.chainA.GetContext())
+	ports := suite.chainB.GetSimApp().ICAHostKeeper.GetAllPorts(suite.chainB.GetContext())
```

Comment:
Many small updates to tests, a lot of which are subtle and require functions to just be called on the appropriate chain (chainB for the host submodule in most cases)